src/main/webapp/app/shared/sidebar/sidebar-card-item/sidebar-card-item.component.html

```diff
@@ -51,7 +51,7 @@
                     @if (sidebarItem.icon) {
                         <fa-icon [fixedWidth]="true" [icon]="sidebarItem.icon" />
                     }
-                    {{ sidebarItem.title }}</span
+                    {{ removeChannelPrefix(sidebarItem.title) }}</span
```

Comment:
Please avoid using function calls in the html template. `removeChannelPrefix` also sounds communication specific, which is why this should rather happen in the communication component beforehand and not within the generic `sidebar-card-item.component`.